docs/notebooks/ensembles.ipynb

```diff
@@ -164,6 +164,7 @@
     "# Set display to HTML style (for fancy output)\n",
```

Comment:
<p>Quelques détails de syntaxe ou typo:</p><ul><li><code>surimposed</code> --&gt; <code>superimposed</code>   </li><li>change significance : whether most of the ensemble members project <strong>a statistically significant climate change signal, </strong>in comparison to their internal variability.</li><li>There are<s> be</s> 5 different members in the ensemble</li></ul><p> </p><blockquote>(models agree and all show significant change)</blockquote><p>Je dirais plutôt <code>enough show a significant change</code>. L'IPCC utilise un seuil de X% (60?).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1514/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>